pages/linux/systemctl.md

```diff
@@ -18,6 +18,10 @@
 
 `systemctl enable/disable {{unit}}`
 
+- Mask/Unmask a unit, make it impossible to start it:
```

Comment:
It can prevent the service to be started on bootup in advance.For example, if you don't want tomcat start automaticlly, you can use `systemctl mask tomcat.service`(it will ln -s '/dev/null' '/etc/systemd/system/tomcat.service),then `systemctl enable tomcat.service` can not take effect. Sometimes, two services boot automaticlly both time will conflict, and we can mask one.